android/app/src/main/java/com/coboltforge/dontmind/multivnc/VncCanvasActivity.java

```diff
@@ -329,7 +329,14 @@ public boolean onTouchEvent(MotionEvent e) {
 
 				e = vncCanvas.changeTouchCoordinatesToFullFrame(e);
 
-				vncCanvas.processPointerEvent(e, true, false);
+				// set ACTION according to stylus ACTION (is working on Samsung Galaxy Tab S4) and activate rightButton
+				boolean rightButton = false;
+				if(e.getAction() >= 211 && e.getAction() <= 213){
```

Comment:
Are you able to supply constants instead of those magic numbers?

Reply:
Found https://stackoverflow.com/questions/63533574/getactionmasked-from-motionevent-returns-actionmove-masked-with-some-other-val and https://forum.nomadsculpt.com/t/issue-with-pen-android/243/47